xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/jackson/modules/DidUriSerializer.java

```diff
@@ -0,0 +1,46 @@
+package org.xrpl.xrpl4j.model.jackson.modules;
+
+/*-
+ * ========================LICENSE_START=================================
+ * xrpl4j :: model
+ * %%
+ * Copyright (C) 2020 - 2022 XRPL Foundation and its contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdScalarSerializer;
+import org.xrpl.xrpl4j.model.transactions.DidUri;
+
+import java.io.IOException;
+
+/**
+ * Custom Jackson serializer for {@link DidUri}s.
+ */
+public class DidUriSerializer extends StdScalarSerializer<DidUri> {
+
+  /**
+   * No-args constructor.
+   */
+  public DidUriSerializer() {
+    super(DidUri.class, false);
+  }
+
+  @Override
+  public void serialize(DidUri didUri, JsonGenerator gen, SerializerProvider provider) throws IOException {
+    gen.writeString(didUri.value());
```

Comment:
Should this data actually be hex-encoded? The spec shows this as a `string` in JSON, but in rippled it's just a `BLOB`.

Reply:
See comment above